query/influxql/group.go

```diff
@@ -242,7 +242,7 @@ func (gr *groupInfo) createCursor(t *transpilerState) (cursor, error) {
 
 	// If a function call is present, evaluate the function call.
 	if gr.call != nil {
-		c, err := createFunctionCursor(t, gr.call, cur, !gr.selector || interval > 0)
```

Comment:
I think this will break aggregates. Aggregates should always be normalized.

Reply:
Yeah, I am working on stddev and realized it. Will continue to change this part.